PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -807,6 +810,12 @@ Bool_t AliAnalysisTaskJetExtractor::Run()
   if(fRandomGenerator->Rndm() >= fEventPercentage)
     return kFALSE;
 
+  if (fRejectTPCPileup)   {
+      Bool_t acceptEventCuts = fAliEventCuts.AcceptEvent(InputEvent());
+      if(!acceptEventCuts)     return kFALSE;
+    }
+
+
```

Comment:
You still don't need this line. It's just repeating what's already done automatically: https://github.com/alisw/AliPhysics/blob/80bac393b634e1e5f344a0c0e71cdb2eb379b2f0/PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.cxx#L1276